src/lib/crypto_wrappers/x509/Certificate.ts

```diff
@@ -268,9 +271,11 @@ export default class Certificate {
       throw new CertificateError(verification.resultMessage);
     }
 
-    return verification.certificatePath.map(
-      (pkijsCert: pkijs.Certificate) => new Certificate(pkijsCert),
-    );
+    return verification.certificatePath
```

Comment:
Can `CertificateChainValidationEngineVerifyResult.certificatePath` really be `undefined`? I've just checked the code and it seems like it's always set:

https://github.com/PeculiarVentures/PKI.js/blob/ecf54a6991380e702afa44a7099e876b951115f6/src/CertificateChainValidationEngine.ts#L1459

Just asking because CI is failing because we don't have a test covering the case where `certificatePath === undefined` so I was trying to write one, but can't figure out how.